### PR TITLE
Issue 74 Updates

### PR DESCRIPTION
**Pull request by benasher44**
Ready to go!

### CODE REVIEW

File: _posts/2017-06-15-issue-74.md [hunk 1]
```diff
@@ -39,6 +39,8 @@ Nate Cook made [lots of documentation revisions](https://github.com/apple/swift/
 
 Dave Abrahams opened a PR with some [performance optimizations for `String`](https://github.com/apple/swift/pull/10223).
 
+Roman Levenstein [cherry-picked a patch](https://github.com/apple/swift/pull/10263) from master to the Swift 4 branch that reduces the standard library code size by about 5%-6%.
+
```
Comment (garricn) [inline _posts/2017-06-15-issue-74.md]: 😁

**Comment by jessesquires** [comment]
Thanks @benasher44 ! Looks great 😎 

Just added a few more things 👌

**Comment by BasThomas** [comment]
And by @garricn I mean @benasher44. Oops 😬

**Comment by BasThomas** [comment]
Not really sure why Danger is failing, but I think we should be good to go publishing this. @benasher44, I don't think you can do so right now (as the build is failing), right? Let me know when its done and I'll merge. :)

**Comment by jessesquires** [comment]
@BasThomas 💯  we should wait like 9 more hours before merging 🤣 #timeZones

**Comment by BasThomas** [comment]
Just in case: #271

**Comment by benasher44** [comment]
Did some debugging. Some text in @jessesquires's most recent edit exposed a bug in prose lint

**Comment by benasher44** [comment]
For future reference, more than 1 emoji in a row appears to break danger-prose 😭

**Comment by benasher44** [comment]
I tried making a small repro case with only a few emoji in a row, and that didn't break it. That said, removing the extra emoji did fix it for this article.

**Comment by jessesquires** [comment]
🙌

**Pull request merged**